### PR TITLE
[Docs] mux-uploader and MuxUploader usage and styling instructions

### PR DESCRIPTION
**Pull request by clearlyTHUYDOAN**
## Description

Noticed some outdated docs. Updated them. 🌈 

Also improved the instructions for users trying to get set up for the first time. ✨

### CODE REVIEW

File: packages/mux-uploader-react/README.md [hunk 1]
```diff
@@ -48,8 +48,8 @@ const MuxUploaderWithMuxUploaderDropExample = () => {
   return (
     <div>
       <h1>Simple MuxUploader and Mux Uploader Drop Examples</h1>
-      {/* Upload button by itself. Displays upload progress in text as percentage. */}
-      <MuxUploader url="authenticated-url" type="bar" status></MuxUploader>
+      {/* Roudned upload button by itself. Displays upload progress in text as percentage. */}
```
Comment (cjpillsbury) [inline packages/mux-uploader-react/README.md:51]: typo(blocker): Roudned.

**Review by cjpillsbury — DISMISSED** [review]
One minor ask. Otherwise, approved.

File: packages/mux-uploader-react/README.md [hunk 2]
```diff
@@ -98,3 +100,31 @@ const MuxUploaderWithMuxUploaderDropExample = () => {
...
+| ------------------------------ | ------------------ | ------------------- | ------------------------------------------------------- | ------------------------------------------------------------------------------------------------- |
+| `--uploader-font-family`       | `font-family`      | `Arial`             | font family of the component                            | Applies to other elements as well: upload status and error status                                 |
+| `--uploader-font-size`         | `font-size`        | `16px`              | font size for text within the component                 | Also applies to `<MuxUploaderDrop>` i.e. overlay text c                                           |
+| `--uploader-background-color`  | `background-color` | `inherit`           | background color of area surrounding the upload         |                                                                                                   |
+| `--button-background-color`    | `background`       | `#fff`              | background color of upload button                       |                                                                                                   |
```
Comment (luwes) [inline packages/mux-uploader-react/README.md:115]: nit (non-blocking): is the css prop here background or background-color?

**Review by luwes — APPROVED** [review]
LGTM

File: examples/vanilla-ts-esm/public/mux-uploader-simple.html [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  </head>
+  <body>
+    <h2>Enter your upload GCS url:</h2>
+    <div>
+      <input type="text" class="direct-upload-url" placeholder="https://storage.googleapis.com/..." />
+    </div>
+
+    <div style="margin-top: 40px; width: 400px;">
+      <mux-uploader-drop mux-uploader="my-uploader">
+        <mux-uploader
```
Comment (luwes) [inline examples/vanilla-ts-esm/public/mux-uploader-simple.html:41]: is this a bit strange or is it just me? should mux-uploader wrap the drop area?

File: packages/mux-uploader-react/README.md [hunk 4]
```diff
@@ -48,8 +48,8 @@ const MuxUploaderWithMuxUploaderDropExample = () => {
   return (
     <div>
       <h1>Simple MuxUploader and Mux Uploader Drop Examples</h1>
-      {/* Upload button by itself. Displays upload progress in text as percentage. */}
-      <MuxUploader url="authenticated-url" type="bar" status></MuxUploader>
+      {/* Rounded upload button by itself. Displays upload progress in text as percentage. */}
+      <MuxUploader url="authenticated-url" type="bar" status style={{ '--button-border-radius': '40px' }}></MuxUploader>
```
Comment (luwes) [inline packages/mux-uploader-react/README.md:52]: is this still right `url` vs `endpoint`? 

why was this renamed, I don't see any reference to endpoint in the Mux video docs to `endpoint`

File: packages/mux-uploader/README.md [hunk 5]
```diff
@@ -55,28 +55,178 @@ If you are using ECMAScript modules, you can also load the `mux-uploader.mjs` fi
 <script type="module" src="https://unpkg.com/@mux/mux-uploader/dist/mux-uploader.mjs"></script>
 ```
 
-## Usage
+# Usage
+
+In these examples `"authenticated-url"` is the `url` property in the response returned from [creating a Direct Upload](https://docs.mux.com/api-reference/video#operation/create-direct-upload). Creating a direct upload happens server-side directly to the Mux API.
+
+The `url` for a Direct Upload looks like `"https://storage.googleapis.com/video..."`.
```
Comment (luwes) [inline packages/mux-uploader/README.md:62]: I would make it more clear that "authenticated-url" is their own url with something like

`https://my-authenticated-url.com/storage?long-ass-url-params`

bad habit but I skim a lot and this would look like it's a real thing `endpoint="authenticated-url"` as attribute if skipping the 

> In these examples `"authenticated-url"` is the `url` property in

File: packages/mux-uploader-react/README.md (hunk 4 above)
Comment (clearlyTHUYDOAN) [reply packages/mux-uploader-react/README.md:52]: Out of date. Will fix.

File: packages/mux-uploader/README.md (hunk 5 above)
Comment (clearlyTHUYDOAN) [reply packages/mux-uploader/README.md:62]: I'll clean this up. The backticks also make me think it's either a variable name or the name of the property itself but we're actually instead referring to the value.

**Comment by clearlyTHUYDOAN** [comment]
Okay! Did some clean up. Good for another looksie @luwes.

Might need someone's guidance on how to update this branch without blowing away the changes we've added. During @dylanjha and I's pairing yesterday, there were merge commits so it's already...kind of complicated.

**Comment by clearlyTHUYDOAN** [comment]
Rebased some murky git history and feeling like a CHAMPION 😎

**Review by luwes — APPROVED** [review]
👍
